mobify/progressive-web-scaffold
# Submit checkout shipping

This PR adds form submission to the checkout shipping page, and refetches shipping methods when the user has entered their country, region and postal code. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1111
 **Linked PRs**: #292 

## Changes
- Updated checkout shipping input names to match desktop
- Refetch shipping methods on postal code blur 
- Submit shipping form data to the back end (parse data into correct format and make API call)
- Switch to payment page on successful response

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to your cart
- navigate to: https://www.merlinspotions.com/checkout/
- confirm the page works as expected
